modin/pandas/dataframe.py

```diff
@@ -99,7 +99,8 @@ class DataFrame(BasePandasDataset):
 
     _pandas_class = pandas.DataFrame
 
-    def __init__(
+    @append_to_docstring(__doc__)
+    def _init(
```

Comment:
Should we remove default values here?

Reply:
yes, this is "internal" method now; removed